src/content/en/tools/puppeteer/get-started.md

```diff
@@ -0,0 +1,19 @@
+project_path: /web/tools/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Learn how to use Puppeteer to automate and test apps in headless Chrome.
+
+{# wf_updated_on: 2018-01-10 #}
+{# wf_published_on: 2018-01-10 #}
+{# wf_blink_components: Internals>Headless #}
+
+# Quick start {: .page-title }
+
+<<_src/_index/getstarted.md>>
+
+<<_src/_index/runtimesettings.md>>
+
+## Next steps {: #next-steps }
```

Comment:
Optional - alternate way to do this is to use step groups. In the `_toc.yaml`, you can add a `step_group: bob` to each item. DevSite will then automagically add a 'Next' and 'Previous' element to the bottom of each page. Check the bottom of [this](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/) for an example.

Reply:
Nice. I'll hold off for now until we bring in more docs where there's an obvious flow.